API.php

```diff
@@ -50,8 +50,20 @@ public function getKeywordContentFromNameId($idSite, $period, $date, $idSubtable
         $dataTable = $this->getDataTable(Archiver::CAMPAIGN_NAME_RECORD_NAME, $idSite, $period, $date, $segment, $expanded = false, $idSubtable);
 
         if ($this->isTableEmpty($dataTable)) {
-            $referrersDataTable = ReferrersAPI::getInstance()->getKeywordsFromCampaignId($idSite, $period, $date, $idSubtable, $segment);
```

Comment:
If this is a proxysite issue, it could (maybe) be fixed by sending a proxy request (by using `Request::processRequest` here, instead of using the API directly. Is there a reason for not doing this?